composer.json

```
@@ -142,7 +142,7 @@
         "phpunit/phpunit": "10.5.3",
         "ramsey/collection": "^1.2",
         "ramsey/uuid": "^4.2.3",
-        "rector/rector": "1.2.6",
+        "rector/rector": "1.2.8",
```

```suggestion
        "rector/rector": "~1.2.8",
```

Maybe we should just lock the version this way? :)

---

we are follow phpstan change :), sometime, it break, for personal repo, that's fine, but for repo with many contributors, I think pinned version is better, better to use commit composer.lock as well imo, and use renovate or dependabot to update so when it break, it can directly be executed/skipped by maintainer :)

---

I updated to use ~1.2.8 anyway :) https://github.com/spiral/framework/pull/1159/commits/59bced6285886eff8a436825c3bb1dc38c770927